src/Stripe.net/Services/Account/AccountCreateOptions.cs

```diff
@@ -1,15 +1,114 @@
 namespace Stripe
 {
+    using System.Collections.Generic;
     using Newtonsoft.Json;
+    using Stripe.Infrastructure;
 
-    public class AccountCreateOptions : AccountSharedOptions
+    public class AccountCreateOptions : BaseOptions, IHasMetadata
     {
+        [JsonProperty("account_token")]
+        public string AccountToken { get; set; }
```

Comment:
Missing docstring here, when there's none I usually make up the sentence (and if you're willing you could fix it internally, or en-jira)
But here it exists per https://stripe.com/docs/api/accounts/create#create_account-account_token

Reply:
I would use the exact string. You can put the link if you want (I don't always bother :p)

> An account token, used to securely provide details to the account.

Reply:
yep you did that already so all good.